android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainViewModel.kt

```diff
@@ -225,6 +244,8 @@ constructor(
       )
       .displayName
 
+  private fun resumeSync() = syncBroadcaster.runSync()
```

Comment:
This one liner private function is not necessary.

Reply:
👍🏾 I've refactored the code to remove it